index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <div class="bg">
+    <h1 id="title">
```

Comment:
[OPTIONALLY] here I would recommend you include the `h1` and `p` tags with `header` tag.

Reply:
Good job using the optional tag 👍. Even if this is an optional change, please do consider the comment i have left above.